build/Version.props

```diff
@@ -8,7 +8,8 @@
     <MajorMinorVersion>$(VersionMajor).$(VersionMinor)</MajorMinorVersion>
     <CliVersionNoSuffix>$(VersionMajor).$(VersionMinor).$(VersionPatch)</CliVersionNoSuffix>
     <CliVersionPrefix>$(CliVersionNoSuffix)-$(ReleaseSuffix)</CliVersionPrefix>
-    <CliBrandingVersion>$(CliVersionNoSuffix) - $(ReleaseSuffix)</CliBrandingVersion>
+    <CliBrandingVersion Condition=" '$(DropSuffix)' != 'true' ">$(CliVersionNoSuffix) - $(ReleaseSuffix)</CliBrandingVersion>
+    <CliBrandingVersion Condition=" '$(CliBrandingVersion)'== '' ">$(CliVersionNoSuffix)</CliBrandingVersion>
```

Comment:
Just a small nit regarding space between variable and `==` operator.

Reply:
Missed it. Thanks!